cmd/flags/flags.go

```diff
@@ -60,19 +72,22 @@ type SharedFlags struct {
 	SpanStorage spanStorage
 	// DependencyStorage defines common settings for Dependency Storage.
 	DependencyStorage dependencyStorage
+	// Logging holds logging configuration
+	Logging logging
 }
 
 // InitFromViper initializes SharedFlags with properties from viper
 func (flags *SharedFlags) InitFromViper(v *viper.Viper) *SharedFlags {
 	flags.SpanStorage.Type = v.GetString(spanStorageType)
 	flags.DependencyStorage.DataFrequency = v.GetDuration(dependencyStorageDataFrequency)
+	flags.Logging.Level = v.GetString(logLevel)
 	return flags
 }
 
 // AddFlags adds flags for SharedFlags
 func AddFlags(flagSet *flag.FlagSet) {
 	flagSet.String(spanStorageType, CassandraStorageType, fmt.Sprintf("The type of span storage backend to use, options are currently [%v,%v,%v]", CassandraStorageType, ESStorageType, MemoryStorageType))
-	flagSet.String(logLevel, "info", "Minimal allowed log level")
+	flagSet.String(logLevel, "info", "Minimal allowed log Level. For more levels see zap logger documentation")
```

Comment:
instead of `zap logger documentation` I'd print the url

Reply:
added but it looks a bit weird. 

Message for conf file: 
```
Configuration file in JSON, TOML, YAML, HCL, or Java properties formats (default none). See spf13/viper for precedence.
```